docs/getting-into-the-code/guidelines-for-setting-up-a-local-workspace/trouble-shooting.md

```diff
@@ -83,3 +83,9 @@ Another indication is following output
 ```text
 java.lang.UnsupportedClassVersionError: org/javamodularity/moduleplugin/ModuleSystemPlugin has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
 ```
+
+## Attempts to open preferences panel freezes application
+
+This is likely caused by improper integration of your OS or Desktop Environment with your password prompting program or password manager. Ensure that these are working properly, then restart your machine and attempt to run the program.
```

Comment:
Is it possible to detail "Ensure that these are working properly"? Maybe, giving an example?

Reply:
We could detail what some of the desired states would be, ie. `If everything works, a password prompt will appear as the program starts, given the keyring your OS uses hasn't already been unlocked.`

The trouble is, it depends on OS what the exact implementation being used is, and therefore how to configure it will vary. I don't have Windows or MacOS personally, so I can only speak for Linux users and state that their `xdg-desktop-portal` settings need to refer to active and valid portal programs they have installed. And that may not be the whole story; my lack of knowledge means I'd need to do further research to confirm or deny that.

Reply:
I think, even these sentences help.

What do you think of:

```markdown
To clarify, in an ideal scenario, a password prompt should appear when the program starts, provided the keyring your OS uses has not already been unlocked. However, the implementation details vary depending on the operating system, which makes troubleshooting more complex.

For Windows and macOS users, specific configurations may differ based on the password management tools and settings used, so ensure your OS's password management system is properly set up and functioning.

For Linux users, ensure that your `xdg-desktop-portal` settings refer to active and valid portal implementations installed on your system. However, there might be other factors involved, so additional research or guidance specific to your distribution may be necessary.
```

Reply:
Looks good to me, we can always change the description if we learn more later.

Should I reopen the issue, or just make a PR for the same issue? Or rather document the change in a new issue?

Reply:
just make another PR, e.g. follow up to ###